app/packs/entrypoints/application.js

```diff
@@ -27,3 +27,10 @@ import 'simple-datatables'
 // const imagePath = (name) => images(name, true)
 
 require('modules/validate-forms')
+
+window.addEventListener("turbolinks:before-cache", function() {
```

Comment:
We use turbo, not turbolinks 😬 I missed that the first time.

Reply:
https://github.com/sul-dlss/happy-heron/pull/1796